### PR TITLE
Added https://www.semantic-web-journal.net/system/files/swj3379.pdf

### PR DESCRIPTION
**Pull request by LongquanJiang**

### CODE REVIEW

**Comment by LongquanJiang** [comment]
#116

File: freebase/simple_question_freebase
Comment (xixi019) [inline freebase/simple_question_freebase:1]: update the simple questions freebase md file in your local branch

**Review by xixi019 — COMMENTED** [review]
looks ok to me

**Comment by DavidBlavid** [comment]
looks good, found no errors

**Pull request merged**